src/WebJobs.Script/Diagnostics/ConsoleTraceWriter.cs

```diff
@@ -1,26 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using Microsoft.Azure.WebJobs.Host;
-
-namespace Microsoft.Azure.WebJobs.Script
-{
-    internal class ConsoleTraceWriter : TraceWriter
-    {
-        public ConsoleTraceWriter(TraceLevel traceLevel) : base(traceLevel)
-        {
-        }
-
-        public override void Trace(TraceEvent traceEvent)
-        {
-            if (Level < traceEvent.Level)
-            {
-                return;
-            }
-
-            Console.WriteLine(traceEvent.Message);
```

Comment:
Where is console logging handled now?

Reply:
Oh good catch. I think I accidentally removed it. There's a default console logger that was wired up previously; I'll re-add that.